src/renderers/pixi/node.ts

```diff
@@ -233,6 +235,22 @@ export class NodeRenderer<N extends Node, E extends Edge>{
             this.nodeContainer.addChildAt(this.iconSprite, this.nodeContainer.children.length - 1)
           }
         })
+      } else if (this.icon?.type === 'imageIcon') {
+        const texture = PIXI.Texture.from(this.icon.image)
```

Comment:
can we cache textures by the this.icon.image value?  similar to what we are doing w/ the FontLoader, e.g.
```ts
const cache: Record<string, PIXI.Texture> = {}

const ImageLoader  = (url: string) => {
  if (cache[url] === undefined) {
    cache[url] = new PIXI.Texture.from(url)
  }
  return new Sprite(cache[url])
}
...
this.iconSprite = ImageLoader(this.icon.image)
```